.buildkite/pipeline.yml

```diff
@@ -0,0 +1,40 @@
+env:
+  BK_QUEUE_NAME: "testqueue" # probably don't need this
+  TESTER_VAULT_VERSION: "1.11.2" # this probably needs to get bumped over time
+  SVC_VAULT_VERSION: "1.11.2" # this probably needs to get bumped over time
+  VAULT_ADDR: "http://vault-svc:8200"
+  BUILDKITE_PIPELINE_SLUG: "my_pipeline"
+  DUMP_ENV: "true"
+  VAULT_DEV_ROOT_TOKEN_ID: "88F4384B-98E9-4AE3-B00C-F55678F89080"
+steps:
+  - label: ":sparkles: Lint"
+    plugins:
+      - docker-compose#v4.5.0:
+          run: lint
+
+  - label: ":shell: Shellcheck"
+    plugins:
+      - docker-compose#v4.5.0:
+          run: shellcheck
+
+  - label: ":bat: :test_tube: BATS tests"
+    plugins:
+      - docker-compose#v4.5.0:
+          run: tests
+          
+  - label: ":vault: :test_tube: Integration Tests"
+    command:  /app/.buildkite/steps/test_envvar.sh
+    plugins:
+      docker-compose#v4.5.0:
+        config:
+          - docker-compose-integration.yml
+        run: vault-tester
+        env:
+          - VAULT_ADDR=\${VAULT_ADDR}
+          - BUILDKITE_PLUGIN_VAULT_SECRETS_SERVER=\${VAULT_ADDR}
+          - BUILDKITE_PIPELINE_SLUG=\${BUILDKITE_PIPELINE_SLUG}
+          - BUILDKITE_PLUGIN_VAULT_SECRETS_DUMP_ENV=\${DUMP_ENV}
+          - VAULT_DEV_ROOT_TOKEN_ID=\${VAULT_DEV_ROOT_TOKEN_ID}
+          - BUILDKITE_PLUGIN_VAULT_SECRETS_AUTH_TOKEN=\${VAULT_DEV_ROOT_TOKEN_ID}
+          - SVC_VAULT_VERSION=\${SVC_VAULT_VERSION}
+          - TESTER_VAULT_VERSION=\${TESTER_VAULT_VERSION}
```

Comment:
Most of these variables are already defined in the docker compose file so I don't even think they are really necessary to specify here

Reply:
Agreed - cleaned this up.